src/rtextures.c

```diff
@@ -353,6 +363,14 @@ Image LoadImageSvg(const char *fileNameOrString, int width, int height)
         {
             fileData = LoadFileData(fileNameOrString, &dataSize);
             isSvgStringValid = true;
+
+            // make sure that fileData is a NULL terminated String
+            // as required by nsvgParse()
+            if (fileData[dataSize-1] != '\0')
```

Comment:
What are the changes that an Svg string does not finish with ' \0' ?

Reply:
A svg that is pre compiled into the program
aka: `char* fileSVG = "<svg`...
will have absolutely no problem, as internal strings are automatically NULL terminated by the compiler

however, actual files, not so much.
I managed to test Inkscape and Krita, both open source
neither outputs files without an end byte

Given that .svg is technically a XML file type, I don't think a EOF byte is in the specification